statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -108,8 +102,7 @@ func (psh *PresenterStatusHandler) GetTotalRewardsValue() (string, string) {
 
 // CalculateRewardsPerHour will return an approximation of how many ERDs a validator will earn per hour
 // Rewards estimation per hour will be equals with :
-// (changeToBeInConsensus - changeToBeLeader) * roundsPerHour * hitRate * communityPercentage * Rewards +
-// changeToBeLeader * roundsPerHour * hitRate * leaderPercentage * Rewards
+// changeToBeInConsensus * roundsPerHour * hitRate * denominationCoefficient * Rewards
 func (psh *PresenterStatusHandler) CalculateRewardsPerHour() string {
 	consensusGroupSize := psh.getFromCacheAsUint64(core.MetricConsensusGroupSize)
```

Comment:
func too long. Might consider having some calls like
chanceToBeInConsensus = computeChanceToBeInConsensus(...)
currentRoundsPerHour = computeRoundsPerHourAccordingToHitRate(...)
erdRewards = computeRewardsInErd(...)
And then multiply them. Also  computeRewardsInErd(...) cand also be used in GetTotalRewardsValue